src/common/kmgr_utils.c

```diff
@@ -94,7 +103,8 @@ kmgr_wrap_key(uint8 *key, const uint8 *in, int insize, uint8 *out)
 	int outsize;
 
 	if (!keywrap_initialized)
-		initialize_keywrap_ctx();
+		if (!initialize_keywrap_ctx())
+			return false;
```

Comment:
I think we should raise an error if initialization failed. Otherwise the caller of `kmgr_wrap_key` cannot know the problem is in either  initialization or wrapping a key.